shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -158,7 +160,15 @@ public AccessibilityNodeInfo createAccessibilityNodeInfo(int virtualViewId) {
         }
 
         if (object.children != null) {
-            for (SemanticsObject child : object.children) {
```

Comment:
there's a todo a bit higher up that might be obsolete now

Reply:
Removed that comment and added an RTL-TODO to the new code.